app/assets/config/manifest.js

```diff
@@ -1,2 +0,0 @@
-//= link_tree ../images
```

Comment:
WHERE IS CODDEEEEEE

Reply:
It's all gone, but you can always get a new job